### ISSUE TITLE
Enable LBaaS neutron plugin

### ISSUE THREAD

**Issue by larsks** [CCI-MOC/esi#680]
We could really use a load balancer in ESI. To expose openshift nodeport services via the existing "floating ip port forwarding" mechanism, we are only able to forward to a single worker node. If this node goes offline, we lose access to all our services (which would probably render all hosted clusters unusable). A load balancer would allow us to forward that traffic to "all worker nodes" (or a subset thereof), so that we can offer a more robust service.

**Comment by tzumainn** [comment]
It looks like the LBaaS plugin was deprecated in RHOS (https://docs.redhat.com/en/documentation/red_hat_openstack_platform/13/html/networking_guide/sec-lbaas#network-lbaas-topo); the replacement is Octavia, a full-fledged service which I'd be reluctant to deploy on current ESI. I think it may still be possible to enable the LBaaS plugin, but I don't know how supported it will be.

A question though - shouldn't this sort of failover handled in the OpenShift architecture? If we move towards the ACM/baremetal-operator model, I don't think we'd be able to take care of this use case in OpenStack.

**Comment by larsks** [comment]
> It looks like the LBaaS plugin was deprecated in RHOS

Yeah, I was [just reading about that](https://wiki.openstack.org/wiki/Neutron/LBaaS/Deprecation#What_are_the_replacement_projects.3F).
 
> A question though - shouldn't this sort of failover [be] handled in the OpenShift architecture?

Load balancers at the network layer are an extremely common bit of infrastructure -- we're making use of them for effectively all of our public-facing openshift clusters. Having something like this available in ESI would be incredibly useful, and not just for OpenShift.

It's possible to solve this in OpenShift instead, but having this feature at the network layer is substantially simpler.

**Comment by tzumainn** [comment]
I think we're winding down development on ESI and concentrating on O-SAC; closing this issue.
